projects/demo/src/modules/components/action-bar/examples/3/index.less

```diff
@@ -0,0 +1,4 @@
+tui-action-bar {
+    top: 1rem;
+    bottom: unset !important;
```

Comment:
it looks like `!important` is unnecessary

Reply:
It would be if we use class instead of a tag selector, since :host inside the component has higher specificity

Reply:
> It would be if we use class instead of a tag selector, since :host inside the component has higher specificity

It works fine without !important